plugins/carla/carla-bridge-wrapper.cpp

```diff
@@ -0,0 +1,558 @@
+/*
+ * Carla plugin for OBS
+ * Copyright (C) 2023 Filipe Coelho <falktx@falktx.com>
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#include "carla-bridge.hpp"
```

Comment:
System header includes should come first, current working directory headers second.